packages/lesswrong/components/sequences/SequenceDraftsList.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { useState } from 'react';
+import { Components, registerComponent } from '../../lib/vulcan-lib';
+import { useLocation } from '../../lib/routeUtil';
+import { useMulti } from '../../lib/crud/withMulti';
+import { useCurrentUser } from '../common/withUser';
+import { sortings } from '../posts/DraftsList';
+
+const styles = (theme: ThemeType): JssStyles => ({
+  item: {
+    listStyle: "none",
+    position: "relative",
+    padding: 5,
+    cursor: "pointer",
+    minWidth: 400,
+  },
+  [theme.breakpoints.down('xs')]: {
+    item: {
+      minWidth: 230,
+    },
+  }
+})
+
+const SequenceDraftsList = ({limit, title="My Drafts", userId, classes, addDraft, dialogPostIds}: {
+  classes: ClassesType,
+  limit: number,
+  title?: string,
+  userId?: string,
+  addDraft: Function,
+  dialogPostIds: string[],
+}) => {
+  const [showSettings, setShowSettings] = useState(false);
+  const currentUser = useCurrentUser();
+  const { Loading } = Components
+  
+  const { query } = useLocation();
+  
+  const terms: PostsViewTerms = {
+    view: "drafts",
+    userId: userId ?? currentUser?._id,
+    limit,
+    sortDraftsBy: query.sortDraftsBy ?? query.view ?? currentUser?.draftsListSorting ?? "lastModified",
+    includeArchived: !!query.includeArchived ? (query.includeArchived === 'true') : currentUser?.draftsListShowArchived,
+    includeShared: !!query.includeShared ? (query.includeShared === 'true') : (currentUser?.draftsListShowShared !== false),
+  }
+  
+  const { results, loading, loadMoreProps } = useMulti({
+    terms,
+    collectionName: "Posts",
+    fragmentName: 'PostsList',
+    fetchPolicy: 'cache-and-network',
+    nextFetchPolicy: "cache-first",
+  });
+
+  if (!currentUser) return null
+
+  const currentSorting = terms.sortDraftsBy || "lastModified"
```

Comment:
I don't think the `|| "lastModified"` is necessary, since that's already happening in the declaration of `terms`

Reply:
Maybe this can just be declared (above `terms`) as `const currentSorting = query.sortDraftsBy ?? query.view ?? currentUser?.draftsListSorting ?? "lastModified";`, and then `terms` just has `sortDraftsBy: currentSorting`?

Reply:
I've made this change in SequenceDraftsList.tsx, and also in DraftsList.tsx (the "My Drafts" component, upon which SequenceDraftsList is based).